CONTRIBUTING.md

```diff
@@ -74,7 +75,7 @@ Pages are associated with a layout depending on their path in the Jekyll configu
 
 Pages are transformed depending on their format (markdown…), then a templating langage, [Liquid](https://shopify.github.io/liquid/), is used to allow some programming (variables, conditionals, loop…). Special variables like `site` and `page` are available to access frontmatter and site data.
 
-If this is not sufficient then you may use HAML Ruby blocks. The Liquid special variables are available in HAML includes but _not_ pages. They can be used identically as in Liquid. To make an include you need to create your HAML content in `source/_includes/<include>.haml` and reference if in your page using: `{% haml <include>.haml %}` (with <include> a name of your choice).
+If this is not sufficient then you can use HAML Ruby blocks. The Liquid special variables are available in HAML includes but _not_ pages. They can be used identically as in Liquid. To make an include you need to create your HAML content in `source/_includes/<include>.haml` and reference if in your page using: `{% haml <include>.haml %}` (with <include> a name of your choice).
```

Comment:
Your change is fine but I have some suggestions/comments about the legacy text:
1 .commas after "sufficient"  and "To make an include".
2. Is "identically" the correct word? I can't think of a replacement, but it feels a bit "off" to me. If you can think of a better word, please use it.
3. Last sentence replace "if" with "it."  Suggestion "To make an include, create HAML content in .."

Reply:
1. I left as-is.
2. I rewrote the sentence a bit.
3. Fixed.